src/FsAutoComplete.Core/DocumentationFormatter.fs

```diff
@@ -16,7 +16,22 @@ module DocumentationFormatter =
 
   let mutable lastDisplayContext: FSharpDisplayContext = FSharpDisplayContext.Empty
 
-  let emptyTypeTip = [||], [||], [||], [||], [||], [||]
+  type EntityInfo =
+    { Constructors: string array
+      Fields: string array
+      Functions: string array
+      Interfaces: string array
+      Attributes: string array
+      DeclaredTypes: string array }
+
+    static member Empty =
+
+      { Constructors = [||]
+        Fields = [||]
+        Functions = [||]
+        Interfaces = [||]
+        Attributes = [||]
+        DeclaredTypes = [||] }
```

Comment:
This change alone would be a wonderful PR - thank you for taking the time to make this nicer!

Reply:
Ah ah, yes having a record instead of the tuple make it so much easier to understand what you have in hand.

I was getting lost with stuff like `string * (string * string []) * (string * string * string * string * string * string)` etc.